skills/odl-pdf/SKILL.md

```
@@ -0,0 +1,740 @@
+
+converter = PdfConverter(options)
+
+# Process all files in a single batch call — avoids multiple JVM startups
+results = converter.convert(["file1.pdf", "file2.pdf", "file3.pdf"])
+
+for result in results:
+    print(result.markdown)
+```
+
+**Node.js:**
+```javascript
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)